scripts/deploy_e2e_testing_infrastructure.sh

```diff
@@ -16,6 +16,7 @@ fi
 
 if ! aws cloudformation describe-stacks --stack-name DeadlineCloudAgentE2EInfrastructure 2>&1 > /dev/null; then
   OP=create-stack
+  echo "Creating the DeadlineCloudAgentE2EInfrastructure stack as it does not already exist"
```

Comment:
This is because currently it prints an error when not finding the stack, which may lead to thinking that the script had an error.